election-config/service.bal

```diff
@@ -0,0 +1,74 @@
+import election_config.elections;
+
+import ballerina/http;
+import ballerina/persist;
+
+listener http:Listener ElectionConfigListener = new (8080);
+elections:Client dbElection = check new ();
+
+service /electionConfig/api/v1 on ElectionConfigListener {
+
+    resource function get elections() returns elections:Election[]|error {
+        stream<elections:Election, persist:Error?> electionStream = dbElection->/elections;
+        elections:Election[] elections = check from elections:Election election in electionStream
+            select election;
+
+        return elections;
+    }
+
+    resource function get elections/[string electionId]() returns elections:Election|error {
+
+        elections:Election|persist:Error election = check dbElection->/elections/[electionId];
+        return election;
```

Comment:
It might be better to handle the scenario of `election` being an `error` instead of blindly returning it all